### PR TITLE
MOL-301 - Add functional test for all topics from MOL-288 task

### PR DESCRIPTION
**Pull request by senghe**

### CODE REVIEW

File: tests/PHPUnit/Functional/Api/RefundOrderWebhookTest.php [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+
+    /** @var CreditMemoRepositoryInterface */
+    private $creditMemoRepository;
+
+    /** @var EntityManagerInterface */
+    private $entityManager;
+
+    public function setUp(): void
+    {
+        parent::setUp(); // TODO: Change the autogenerated stub
```
Comment (marekrzytki) [inline tests/PHPUnit/Functional/Api/RefundOrderWebhookTest.php:52]:
```suggestion
        parent::setUp();
```

File: tests/PHPUnit/Functional/Api/RefundOrderWebhookTest.php [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+use Sylius\RefundPlugin\Entity\LineItemInterface;
+use Sylius\RefundPlugin\Entity\RefundInterface;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Tests\BitBag\SyliusMolliePlugin\PHPUnit\Functional\FunctionalTestCase;
+
+final class RefundOrderWebhookTest extends FunctionalTestCase
+{
+    /** @var MollieApiClient */
+    private $mollieApiClient;
```
Comment (marekrzytki) [inline tests/PHPUnit/Functional/Api/RefundOrderWebhookTest.php:27]: Could You change all fields to typed properties?

**Review by marekrzytki — CHANGES_REQUESTED** [review]

File: tests/PHPUnit/Functional/Api/RefundOrderWebhookTest.php (hunk 1 above)
Comment (senghe) [reply tests/PHPUnit/Functional/Api/RefundOrderWebhookTest.php:52]: Done :)

File: tests/PHPUnit/Functional/Api/RefundOrderWebhookTest.php (hunk 2 above)
Comment (senghe) [reply tests/PHPUnit/Functional/Api/RefundOrderWebhookTest.php:27]: Done :)

**Review by marekrzytki — APPROVED** [review]

**Pull request merged**